app/views/groups/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="add-cat-nav">
+ <%= link_to  groups_path, class: "link-add-cat ps-2" do %>
+    <i class="fa-sharp fa-solid fa-arrow-left top-link"></i>
+  <% end %>
+  <h3 class="nav-add-cat">DETAILS</h3>
+</div>
+
+<div class="container-fluid">
+  <%= render "group", group: @group%>
+
+  <div class="container-fluid">
+  <ul style="list-item-style:none; margin-left:0px;padding-left:0px;">
+    <%@purchases.each do |purchases|%>
+    <li class= "category-item d-flex flex-row justify-content-between align-items-center p-2 mt-3 card-">
+      <p class="ps-2"><%= purchases.name %></p>
+      <p class="pe-2">$<%= purchases.amount %></p>
+    </li>
+    <%end%>
+  </ul>
+  <div class="d-flex justify-content-center btn-p">
+    <%= link_to "New Purchase", new_group_purchase_path(@group), class: "btn-prima" %>
+  </div>
+  </div>
+</div>
```

Comment:
- [ ] also as i mentioned in the index page  the button to add purchase is at the bottom of the page, kindly make your page to look like the design given
your design :point_down: 
![Screenshot from 2023-04-28 11-47-07](https://user-images.githubusercontent.com/66097371/235131496-4a5ed5b2-03a3-4ce8-b970-b2bd6aff786e.png)


given design :point_down: 
![Screenshot from 2023-04-28 11-55-45](https://user-images.githubusercontent.com/66097371/235130758-be3ff113-024c-42b0-824e-989696f2a579.png)